index.yaml

```diff
@@ -4187,15 +4187,15 @@ entries:
       catalog.cattle.io/os: linux
       catalog.cattle.io/release-name: system-upgrade-controller
     apiVersion: v1
-    appVersion: v0.7.3
-    created: "2021-08-06T16:57:34.437773827-07:00"
+    appVersion: v0.7.4
+    created: "2021-08-20T12:01:13.903713-07:00"
     description: General purpose controller to make system level updates to nodes
-    digest: eab5634baf554810c27f81a61f7267d10d1723cd456b15eb335d0a6e01104280
+    digest: 8462ac1f98dd8159ec7cfe61b75701329e41e61b98af87c34fe3ca4515344ecb
     home: https://github.com/rancher/system-charts/charts/system-upgrade-controller
     name: system-upgrade-controller
     sources:
     - https://github.com/rancher/system-charts/charts/system-upgrade-controller
     urls:
-    - assets/system-upgrade-controller/system-upgrade-controller-100.0.0.tgz
-    version: 100.0.0
-generated: "2021-08-06T16:57:33.967930468-07:00"
+    - assets/system-upgrade-controller/system-upgrade-controller-100.0.0+up0.3.0.tgz
+    version: 100.0.0+up0.3.0
+generated: "0001-01-01T00:00:00Z"
```

Comment:
nit: this seems really weird. Why did this timestamp get generated?

I don't think it makes a difference though w.r.t. charts, so I will not block this PR on it.

Reply:
I deleted the full blob of system-upgrade-controller in index.yaml and ran `make charts` again. Should I keep it as is?

Reply:
yeah keep it as-is